app/search_builders/curation_concerns/README.md

```diff
@@ -0,0 +1,69 @@
+# Search Builders
+
+Building searches is core to any Blacklight app, and CurationConcerns is no exception.  
+This directory contains our Search Builders, so named because the initial design followed a builder pattern, meaning
+that when invoked to set values, methods return the object itself, so that invocations can be chained, like:
+
+```ruby
+builder = Blacklight::SearchBuilder.new(processor_chain, scope)
+            .rows(20)
+            .page(3)
+            .with(q: 'Abraham Lincoln')
+```
+
+However, at this level, many if not most of the additional methods do not follow this pattern.  
+Refer to the `Blacklight::SearchBuilder` class if you want to be certain.  That leads to the next topic...
+
+## Ancestry
+
+Most of the SearchBuilders have `::SearchBuilder` as a parent or ancestor.  `::SearchBuilder` does not exist in any repo: it is generated
+by Blacklight and modified by CurationConcerns.  Others descend from `Blacklight::SearchBuilder`, or various other relatives.  
+
+### ::SearchBuilder
+
+The generated parent class `SearchBuilder` descends from `Blacklight::SearchBuilder`.
+As modified by CurationConcerns' installer, it includes additional modules and overrides.  So if your SearchBuilder has `::SearchBuilder` as a parent class, you are getting:
+- [Blacklight::SearchBuilder](https://github.com/projectblacklight/blacklight/blob/master/lib/blacklight/search_builder.rb) grandparent class
+- [Blacklight::Solr::SearchBuilderBehavior](https://github.com/projectblacklight/blacklight/blob/master/lib/blacklight/solr/search_builder_behavior.rb) associated methods
+- [Hydra::AccessControlsEnforcement](https://github.com/projecthydra/hydra-head/blob/master/hydra-access-controls/lib/hydra/access_controls_enforcement.rb) module
+  -  [Blacklight::AccessControls::Enforcement](https://github.com/projectblacklight/blacklight-access_controls/blob/master/lib/blacklight/access_controls/enforcement.rb) ancestor of `Hydra::AccessControlsEnforcement`
+- [CurationConcerns::SearchFilters](https://github.com/projecthydra/curation_concerns/blob/master/app/search_builders/curation_concerns/search_filters.rb)  module that itself includes:
+  - [BlacklightAdvancedSearch::AdvancedSearchBuilder](https://github.com/projectblacklight/blacklight_advanced_search/blob/master/lib/blacklight_advanced_search/advanced_search_builder.rb) more magic for compound Boolean queries
+  - [CurationConcerns::FilterByType](https://github.com/projecthydra/curation_concerns/blob/master/app/search_builders/curation_concerns/filter_by_type.rb) Collection vs. Work filtering, specifically the `filter_models` method
+
+Seem like lot?  Well, this is the simplified view.
```

Comment:
Ah, no. That was just to acknowledge the complexity and also that this is not a comprehensive list.  I can rework it to that effect.